recipe/meta.yaml

```diff
@@ -11,21 +11,20 @@ source:
   sha256: 69e9fa08bacb2475e636da05a6c222b17c67f1ebeab3793762062248dd9d842f
 
 build:
-  number: 201
+  number: 202
   features:
     - blas_{{ variant }}
   skip: true  # [win]
 
 requirements:
   build:
     - toolchain
-    - gcc
+    - gcc  # [unix]
     - cmake
     - blas 1.1 {{ variant }}
     - openblas 0.2.19|0.2.19.*
 
   run:
-    - libgfortran
```

Comment:
Sure, but what if one day it stopped doing that for whatever reason. Then this package would be broken. It may be unlikely that this ever happens, but still it is good to explicitly declare dependencies instead of implicitly declaring them.